{"uuid": "88fc5614-8e14-4c74-8db3-fe6231778515", "counter": 7, "stdout": "\u001b[1;31mfatal: [192.168.1.1]: UNREACHABLE! => {\"changed\": false, \"msg\": \"Failed to connect to the host via ssh: ssh: connect to host 192.168.1.1 port 22: Connection refused\", \"unreachable\": true}\u001b[0m", "start_line": 5, "end_line": 6, "runner_ident": "26dc3f06-ec2a-4b3a-bb74-0fd47f89221d", "event": "runner_on_unreachable", "pid": 7189, "created": "2022-04-22T13:30:49.080453", "parent_uuid": "9e73ed77-7fb2-c452-74fc-00000000000f", "event_data": {"playbook": "deploy.yaml", "playbook_uuid": "fbed95cc-7219-4476-9afb-f48f0fdc05dc", "play": "this is a  hello world example", "play_uuid": "9e73ed77-7fb2-c452-74fc-000000000007", "play_pattern": "all", "task": "Gathering Facts", "task_uuid": "9e73ed77-7fb2-c452-74fc-00000000000f", "task_action": "gather_facts", "task_args": "", "task_path": "/Users/devops/data/project/python/pgoops/upload/iac/helloworld/project/deploy.yaml:2", "host": "192.168.1.1", "remote_addr": "192.168.1.1", "start": "2022-04-22T13:30:49.060515", "end": "2022-04-22T13:30:49.080352", "duration": 0.019837, "res": {"unreachable": true, "msg": "Failed to connect to the host via ssh: ssh: connect to host 192.168.1.1 port 22: Connection refused", "changed": false}, "uuid": "88fc5614-8e14-4c74-8db3-fe6231778515"}}